WalletWasabi.Gui/CommandLine/Daemon.cs

```diff
@@ -68,14 +70,14 @@ internal async Task RunAsync(string walletName, string destinationWalletName, bo
 
 				Logger.LogInfo("Correct password.");
 
-				await Global.InitializeNoWalletAsync();
-				if (Global.KillRequested)
+				await Global.InitializeNoWalletAsync(TerminateService);
```

Comment:
I don't see any `ConfigureAwait(false)` in this file. Is that by design?

Reply:
I don't think so.